app/process-batches/quarantine-file.js

```diff
@@ -0,0 +1,10 @@
+const blobStorage = require('../blob-storage')
+const sendBatchQuarantineEvent = require('../event/send-batch-quarantine-event')
+
+const quarantineFile = async (filename, error) => {
+  console.error(`Quarantining ${filename}, failed to parse file`, error)
```

Comment:
This function has the same signature as in other repos, however in this, there are v call outs tht acc'ly provide an `error` object.


Also, if we're expecting `error` objects as opposed to string, should this logging be changed to better log JS objects?

Reply:
I'd remove the error parameter as it's not used as you say.  Also looking at the other functions they are logging messages detailing the reason why the file was quarantined.